node/full_node_integration_test.go

```diff
@@ -210,131 +121,59 @@ func TestFullNodeTestSuite(t *testing.T) {
 	suite.Run(t, new(FullNodeTestSuite))
 }
 
-func (s *FullNodeTestSuite) TestSubmitBlocksToDA() {
-	require := require.New(s.T())
+func (s *FullNodeTestSuite) TestBlockProduction() {
+	s.executor.InjectTx([]byte("test transaction"))
+	err := waitForAtLeastNBlocks(s.node, 5, Store)
+	s.NoError(err, "Failed to produce second block")
 
-	// Get initial state
-	initialDAHeight := s.node.blockManager.GetDAIncludedHeight()
```

Comment:
daIncludedHeight is not a DA height, it's a block height. This tested the wrong thing.